syntax = "proto3";
package installed_micro_app;




import "model/micro_app/installed_micro_app.proto";

/*
ListMicroApp请求
*/
message ListMicroAppRequest {
    /*
    页码
    */
    int32 page = 1 ;
    /*
    每页大小
    */
    int32 page_size = 2 ;
}

/*
ListMicroApp返回
*/
message ListMicroAppResponse {
    /*
    页数
    */
    int32 page = 1 ;
    /*
    页大小
    */
    int32 page_size = 2 ;
    /*
    总数
    */
    int32 total = 3 ;
    /*
    数据列表
    */
    repeated micro_app.InstalledMicroApp list = 4 ;
}

/*
ListMicroAppApi返回
*/
message ListMicroAppResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    installed_micro_app.ListMicroAppResponse data = 4 ;
}

